meta-intel-edison-distro/conf/distro/poky-edison.conf

```diff
@@ -6,7 +6,7 @@ PREFERRED_PROVIDER_zlib ?= "zlib"
 PREFERRED_PROVIDER_zlib-native ?= "zlib-native"
 PREFERRED_PROVIDER_nativesdk-zlib ?= "nativesdk-zlib"
 # LTS kernel
-PREFERRED_VERSION_linux-yocto = "5.10.63%"
+PREFERRED_VERSION_linux-yocto = "5.15%"
 # test kernel
 #PREFERRED_VERSION_linux-yocto = "5.14%"
 # preempt_rt kernel
```

Comment:
In a later patch we provide for default kernel 5.15.25 LTS, for test kernel  5.16 and for preempt_rt 5.15.25-rt33. This is also a moving target. User can easily (and should) goto latest LTS by just renaming the file linux-yocto_5.15.25.bb to the latest. Of course for Honnister we will release with the latest available LTS at that time.